test/e2e/framework/azure/get-fqdn.go

```diff
@@ -0,0 +1,27 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	armcontainerservice "github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/containerservice/armcontainerservice/v4"
+)
+
+func GetFqdnFn(subscriptionId, resourceGroupName, clusterName string) (string, error) {
```

Comment:
nit: don't really need to suffix function calls with Fn in Go

Reply:
I added that to leave the name `GetFqdn` for a Step that could be created with same goal in the framework